### ISSUE TITLE
Error installing previously working mods

### ISSUE THREAD

**Issue by Zaragata** [NiceneNerd/BCML#22]
Hey! Updated to 2.4.0 through `pip install --upgrade bcml` and decided to reinstall my mods since both BCML and Linkle updated.

Some points:
- I've been using all of the following mods already on 2.3.4
- Made sure all the `.7z` had the same structure `./Mod.7z/Mod/content`
- Cemu is 1.15.15b
- BotW en_US

I installed [Linkle](https://gamebanana.com/skins/163132) and the [dialogue mod](https://gamebanana.com/guis/34282), went fine.

Then I tried to install [this mod](https://gamebanana.com/sounds/39456) to stop the beeps. BCML didn't like it and outputted this:

```
BCML has encountered an error while performing an operation. Error details:

There was an error while processing No Sensor Ringing. This could indicate there is a problem with the mod itself, but it could also reflect a new or unusual edge case BCML does not anticipate. Here is the error:

Traceback (most recent call last):
  File "c:\python37\lib\site-packages\bcml\install.py", line 326, in install_mod
    generate_logs(tmp_dir=tmp_dir, verbose=verbose, options=options)
  File "c:\python37\lib\site-packages\bcml\install.py", line 231, in generate_logs
    merger.log_diff(tmp_dir, modded_files)
  File "c:\python37\lib\site-packages\bcml\texts.py", line 611, in log_diff
    diff_material = self.generate_diff(mod_dir, diff_material)
  File "c:\python37\lib\site-packages\bcml\texts.py", line 600, in generate_diff
    lang_map[lang] = [l for l in LANGUAGES if l in mod_langs][0]
IndexError: list index out of range
```

Then I tried to install the [DS4 UI](https://gamebanana.com/guis/34463) mod. Same output (just to confirm):

```
Traceback (most recent call last):
  File "c:\python37\lib\site-packages\bcml\install.py", line 326, in install_mod
    generate_logs(tmp_dir=tmp_dir, verbose=verbose, options=options)
  File "c:\python37\lib\site-packages\bcml\install.py", line 231, in generate_logs
    merger.log_diff(tmp_dir, modded_files)
  File "c:\python37\lib\site-packages\bcml\texts.py", line 611, in log_diff
    diff_material = self.generate_diff(mod_dir, diff_material)
  File "c:\python37\lib\site-packages\bcml\texts.py", line 600, in generate_diff
    lang_map[lang] = [l for l in LANGUAGES if l in mod_langs][0]
IndexError: list index out of range
```

Then I tried to install [Linkle's icons](https://gamebanana.com/skins/170950) and the [DOF mod](https://gamebanana.com/gamefiles/8472). Same thing.

That's all! Hoping this report serves of something. Thanks ^^

**Comment by Zaragata** [comment]
Looks like it was fixed on 2.4.1 :P

**Comment by Zaragata** [comment]
All working now! :D

![image](https://user-images.githubusercontent.com/28264101/66015821-15051400-e4a2-11e9-8004-a75cb05cf2a8.png)
